network-api/networkapi/wagtailpages/pagemodels/customblocks/text_only_teaser.py

```diff
@@ -1,13 +1,30 @@
+from django.forms.utils import ErrorList
 from wagtail.core import blocks
+from wagtail.core.blocks.struct_block import StructBlockValidationError
 
 
 class TextOnlyTeaserCard(blocks.StructBlock):
 
     heading = blocks.CharBlock(help_text="Heading for the card.")
     heading_link = blocks.PageChooserBlock(required=False, help_text="Page that the header should link out to.")
```

Comment:
What do you think about renaming this field to `link_page` or something like that? I think that would feel a bit more consistent from a naming perspective.

Reply:
Sure thing, added in bf60a832c284edc85749b846ba6e33f76d990d29